src/components/MapInput/index.js

```diff
@@ -115,10 +116,18 @@ const MapInput = ({ className, value, onChange, mapOptions, ...otherProps }) =>
   }, [marker, location, hasLocation]);
 
   useEffect(() => {
-    if (!value?.geometrie?.coordinates && !value?.addressText) return;
+    if (!map) return;
+    if (!value?.location && !value?.addressText) return;
```

Comment:
I don't see `value.addressText` being used in this function. Does it need to be present?

Reply:
removed